visualization/codemirror/sourceCode.js

```diff
@@ -97,7 +97,9 @@ var sourceCode = (function() {
 	 * Clear all three arrays
 	 */
 	function emptyArrays() {
-		currentAsatWarnings, currentCatWarnings, currentMessageWarnings = [];
+		currentAsatWarnings = [];
```

Comment:
Why was this change needed? Is it because you they are referencing to the same array?